server/models/User.js

```diff
@@ -12,13 +12,15 @@ const userSchema = new mongoose.Schema({
     required: true,
     select: false,
   },
+  // Should consider making this camelCase rather than snake_case
```

Comment:
Sure

> Looks good to me, nice fix! @eng-dallavecchia when this is merged to main, can you merge main into #45 as it would work to address one of the comments I left there

Sure!